### PR TITLE
[READY] Gubman II Part 2 - Sniper Rifles & Co.

### PR DESCRIPTION
**Pull request by Draganfrukts**
## About The Pull Request

This is part 2 of Gubman II, it contains some new content and remakes sniper rifles visually (audio too). Also tweaks and rebalances some stuff

## Why It's Good For The Game

More stuff to match the aesthetic overhaul as well as variety.

## Changelog
:cl: Investigator77
tweak: "Sniper Rifle" renamed to "AUS-107 anti-materiel rifle"
tweak: "Syndicate Sniper Rifle" renamed to "'Caracal' anti-materiel rifle"
tweak: "NT-ARG 'Boarder'" renamed to "ARG-63 infantry rifle"
add: Added "SA-107 anti-materiel rifle"
add: Added "MCRS-5B combat rifle", for SolFed.
balance: Syndicate Sniper Rifle now has less recoil and half the firing delay of a normal sniper rifle.
soundadd: New sniper rifle sound
soundadd: New suppressed sniper rifle sound
/:cl:

### CODE REVIEW

**Comment by Draganfrukts** [comment]
Yes the fact that the new syndicate sniper is nicknamed "Caracal" is because the real life counterpart of the sprite is the GM6 "Lynx"

**Comment by Draganfrukts** [comment]
wait i fucked up

**Comment by Draganfrukts** [comment]
I BEG YOU

File: modular_skyrat/modules/aesthetics/guns/code/guns.dm [hunk 1]
```diff
@@ -130,3 +130,73 @@
...
+/obj/item/gun/ballistic/automatic/sniper_rifle
+	name = "AUS-107 anti-materiel rifle"
+	desc = "A long ranged Aussec Armory sniper rifle that does significant damage, features adaptive green hardlight sights instead of a bulky scope."
+	icon = 'modular_skyrat/modules/aesthetics/guns/icons/guns_gubman2.dmi'
+	icon_state = "sniper"
+	fire_sound = 'modular_skyrat/modules/aesthetics/guns/sound/sniperrifle.ogg'
+	suppressed_sound = 'modular_skyrat/modules/aesthetics/guns/sound/sniperrifle_s.ogg'
+	w_class = WEIGHT_CLASS_BULKY
+	can_suppress = FALSE
+/obj/item/gun/ballistic/automatic/sniper_rifle/syndicate
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/aesthetics/guns/code/guns.dm]:
```suggestion

/obj/item/gun/ballistic/automatic/sniper_rifle/syndicate
```

File: modular_skyrat/modules/aesthetics/guns/code/guns.dm [hunk 2]
```diff
@@ -130,3 +130,73 @@
...
+	icon_state = "sysniper"
+	fire_sound = 'modular_skyrat/modules/aesthetics/guns/sound/sniperrifle.ogg'
+	suppressed_sound = 'modular_skyrat/modules/aesthetics/guns/sound/sniperrifle_s.ogg'
+	fire_delay = 20 //Delay halved thanks to recoil absorption
+	burst_size = 0.5
+	recoil = 1
+	can_suppress = TRUE
+	can_unsuppress = TRUE
+	weapon_weight = WEAPON_LIGHT
+/obj/item/gun/ballistic/automatic/sniper_rifle/blackmarket  //New variant of the sniper rifle, adminspawn only for now
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/aesthetics/guns/code/guns.dm]:
```suggestion

/obj/item/gun/ballistic/automatic/sniper_rifle/blackmarket  //New variant of the sniper rifle, adminspawn only for now
```

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]
See above changes.

**Comment by Draganfrukts** [comment]
There we go

**Comment by Draganfrukts** [comment]
Everything tested on localhost, works fine

**Pull request merged**